tfhe/src/integer/server_key/radix/sub.rs

```diff
@@ -416,7 +416,17 @@ impl ServerKey {
         let mut borrow = self.key.create_trivial(0);
         let mut new_blocks = Vec::with_capacity(lhs.blocks.len());
         for (lhs_b, rhs_b) in lhs.blocks.iter().zip(rhs.blocks.iter()) {
-            let mut result_block = self.key.unchecked_sub(lhs_b, rhs_b);
+            let (mut result_block, correction) =
+                self.key.unchecked_sub_with_correcting_term(lhs_b, rhs_b);
```

Comment:
is the unchecked sub behavior correct for trivial zeros ? asking as we are patching things up below and I'm not sure I know how it is supposed to behave

Reply:
Yes behaviour is correct

Reply:
did you check for other places where we had similar patterns to see if the same bug was present ?